.github/workflows/run_terraform.yml

```diff
@@ -0,0 +1,76 @@
+name: "Preparing the cloud environment"
+
+on: 
```

Comment:
I have prepared a trigger that will allow for the parameterization of the workflow:
on: 
  workflow_dispatch:
    inputs:
      instanceType:
        description: 'Instance type'   
        required: true
        type: choice
        options: 
        - t2.nano
        - t2.micro
        - t2.small
        - t2.medium
        - t2.large
        - t2.xlarge
        - t2.2xlarge